src/data/moves/move-attrs/me-first-attr.ts

```diff
@@ -0,0 +1,59 @@
+import type { MoveConditionFunc } from "#app/@types/MoveConditionFunc";
+import type { Move } from "#app/data/moves/move";
+import { CallMoveAttr } from "#app/data/moves/move-attrs/call-move-attr";
+import type { Pokemon } from "#app/field/pokemon";
+import { globalScene } from "#app/global-scene";
+import type { BooleanHolder } from "#app/utils";
+import { BattlerTagType } from "#enums/battler-tag-type";
+import { MoveId } from "#enums/move-id";
+
+/**
+ * Attribute to copy the target's selected (and not yet used)
+ * move for the turn and use it against the target.
+ * Used for {@link https://bulbapedia.bulbagarden.net/wiki/Me_First_(move) | Me First}.
+ * @extends CallMoveAttr
+ */
+export class MeFirstAttr extends CallMoveAttr {
```

Comment:
```suggestion
export class MeFirstAttr extends CallMoveAttr {
  constructor() {
    super();
    this.hasTarget = true;
  }

```
(So that Me First hits the chosen target instead of choosing a random target)